src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsComputeQuota.ps1

```diff
@@ -53,7 +59,7 @@ function New-AzsComputeQuota {
 
         [Parameter(Mandatory = $false)]
         [int32]
-        $CoresLimit = 100,
```

Comment:
Breaking change, add alias to old Parameter value and check to see if using old one and print warning if so.

Reply:
Here is an example, https://github.com/Azure/azure-powershell/blob/3d76129bbd8ac45b6787099a590ee8392f739cf8/src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsSubscriptionsQuota.ps1#L65